lib/Conversion/CGGIToJaxite/CGGIToJaxite.cpp

```diff
@@ -0,0 +1,306 @@
+#include "lib/Conversion/CGGIToJaxite/CGGIToJaxite.h"
+
+#include <cstdint>
+#include <utility>
+
+#include "lib/Conversion/Utils.h"
+#include "lib/Dialect/CGGI/IR/CGGIDialect.h"
+#include "lib/Dialect/CGGI/IR/CGGIOps.h"
+#include "lib/Dialect/Jaxite/IR/JaxiteDialect.h"
+#include "lib/Dialect/Jaxite/IR/JaxiteOps.h"
+#include "lib/Dialect/Jaxite/IR/JaxiteTypes.h"
+#include "lib/Dialect/LWE/IR/LWEDialect.h"
+#include "lib/Dialect/LWE/IR/LWEOps.h"
+#include "llvm/include/llvm/ADT/SmallVector.h"           // from @llvm-project
+#include "llvm/include/llvm/Support/Casting.h"           // from @llvm-project
+#include "mlir/include/mlir/Dialect/Arith/IR/Arith.h"    // from @llvm-project
+#include "mlir/include/mlir/Dialect/Func/IR/FuncOps.h"   // from @llvm-project
+#include "mlir/include/mlir/Dialect/Tensor/IR/Tensor.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/BuiltinTypeInterfaces.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/BuiltinTypes.h"           // from @llvm-project
+#include "mlir/include/mlir/IR/ImplicitLocOpBuilder.h"   // from @llvm-project
+#include "mlir/include/mlir/IR/PatternMatch.h"           // from @llvm-project
+#include "mlir/include/mlir/IR/Types.h"                  // from @llvm-project
+#include "mlir/include/mlir/IR/Value.h"                  // from @llvm-project
+#include "mlir/include/mlir/IR/Visitors.h"               // from @llvm-project
+#include "mlir/include/mlir/Support/LLVM.h"              // from @llvm-project
+#include "mlir/include/mlir/Support/LogicalResult.h"     // from @llvm-project
+#include "mlir/include/mlir/Transforms/DialectConversion.h"  // from @llvm-project
+
+namespace mlir::heir {
+
+#define GEN_PASS_DEF_CGGITOJAXITE
+#include "lib/Conversion/CGGIToJaxite/CGGIToJaxite.h.inc"
+
+class CGGIToJaxiteTypeConverter : public TypeConverter {
+ public:
+  CGGIToJaxiteTypeConverter(MLIRContext *ctx) {
+    addConversion([](Type type) { return type; });
+    // TODO(gshruthi): Is this conversion needed for jaxite?
```

Comment:
Done. Added a check to check if the LweCleartext bitwidth is 3 bits (since we only support lut3 now)

Filed Issue to support types.LwePlaintext and types.LweCiphertext - https://github.com/google/heir/issues/855